support/troubleshooting.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "Frequently Asked Questions"
+---
+
+<AccordionGroup>
+  <Accordion title="I've updated shuttle and my project isn't working anymore.">
+    If you've recently updated shuttle, in order to get your project back up, you need to run:
+    **`cargo shuttle project rm`** followed by
+    **`cargo shuttle project new`** and everything should be back in working order.
+  </Accordion>
+  <Accordion title="I can’t deploy nor run locally, what should I do? (Windows)">
+    Some projects currently fail to deploy on Windows. The bug is related to [this Cargo meta-issue](https://github.com/rust-lang/cargo/issues/9770). Cargo is still trying to determine how to fix them properly in general.
+
+    As a temporary fix, you can try running **`cargo build`** before **`cargo shuttle deploy`** or **`cargo shuttle run`**.
+  </Accordion>
+  <Accordion title="I’m struggling to install shuttle on my M1/M2 Mac, what should I do?">
+    There’s a known bug with arm64 because of the openssl crates being built for x86, rather than arm64. In order to get around that issue, you can either run:
+
+    `**cargo install --features vendored-openssl cargo-shuttle**`
+
+    or install it via binary:
+
+    `**cargo binstall cargo-shuttle**`
+  </Accordion>
+  <Accordion title="How can I return custom errors with shuttle?">
+    CustomError is an alias for **`anyhow::Error`**, the simplest way to output a relevant message here is by using **`anyhow::Context`** like this:
+    
+    ```rust
+        pool.execute(include_str!("../schema.sql"))
+        .await
+        .context("failed to run migrations")?;
+    ```
+  </Accordion>
+  <Accordion title="How can I handle environment variables?">
+    Add **`shuttle-secrets`** to the dependencies for your service, and add a **`Secrets.toml`** to the root of your project with the secrets you'd like to store. Make sure to add **`Secrets.toml`** to a **`.gitignore`** to omit your secrets from version control.
+
+    Next, pass **`#[shuttle_secrets::Secrets] secret_store: SecretStore`** as an argument to your **`shuttle_service::main`** function. **`SecretStore::get`** can now be called to retrieve your API keys and other secrets at runtime.
```

Comment:
Maybe we should include the local secrets here? See the secrets resource docs, I added it there

Reply:
Done!